Octokit.Tests/Clients/RepositoriesClientTests.cs

```diff
@@ -790,5 +790,35 @@ public void GetsCorrectUrl()
                 await Assert.ThrowsAsync<ArgumentException>(() => client.EditBranch("owner", "repo", "", update));
             }
         }
+
+        public class TheSha1Method
+        {
+            [Fact]
+            public async Task EnsureNonNullArguments()
```

Comment:
Minor note that the wording should be "Ensures" rather than "Ensure".  Since the test shoudl read TheGetSha1Method.EnsuresNonNullArguments() :grinning:

Also this test is checking Null and Empty arguments.  In other places ive seen us implement this as 2 test methods eg `EnsuresNonNullArguments()` and `EnsuresNonEmptyArguments()`